### ISSUE TITLE
Build failure with libphonenumber 7.1.0-5+b1

### ISSUE THREAD

**Issue by jberryman** [christian-marie/phone-numbers#4]
After an upgrade to `libphonenumber-dev  7.1.0-5+b1` on debian I'm getting breakage. 

*EDIT*: it looks like the packages that work form me are: `libphonenumber-dev_7.1.0-2_amd64.deb` `libphonenumber7_7.1.0-2_amd64.deb` and `libgeocoding7_7.1.0-2_amd64.deb`.

```
--  While building package phone-numbers-0.0.6 using:
      /home/me/.stack/setup-exe-cache/x86_64-linux/setup-Simple-Cabal-1.22.4.0-ghc-7.10.2 --builddir=.stack-work/dist/x86_64-linux/Cabal-1.22.4.0 build lib:phone-numbers test:test --ghc-options " -ddump-hi -ddump-to-file"
    Process exited with code: ExitFailure 1
    Logs have been written to: /home/me/Work/SignalVine/dev/presta/.stack-work/logs/phone-numbers-0.0.6.log

    Configuring phone-numbers-0.0.6...
    Preprocessing library phone-numbers-0.0.6...
    [1 of 3] Compiling Data.PhoneNumber.FFI ( .stack-work/dist/x86_64-linux/Cabal-1.22.4.0/build/Data/PhoneNumber/FFI.hs, .stack-work/dist/x86_64-linux/Cabal-1.22.4.0/build/Data/PhoneNumber/FFI.o )
    [2 of 3] Compiling Data.PhoneNumber.LowLevel ( lib/Data/PhoneNumber/LowLevel.hs, .stack-work/dist/x86_64-linux/Cabal-1.22.4.0/build/Data/PhoneNumber/LowLevel.o )
    [3 of 3] Compiling Data.PhoneNumber ( lib/Data/PhoneNumber.hs, .stack-work/dist/x86_64-linux/Cabal-1.22.4.0/build/Data/PhoneNumber.o )
    
    In file included from /usr/include/phonenumbers/phonenumberutil.h:34:0:
        0,
                         from cbits/phone-numbers.cc:2:
    
    /usr/include/phonenumbers/phonenumber.pb.h:12:2:
         error: #error This file was generated by a newer version of protoc which is
         #error This file was generated by a newer version of protoc which is
          ^
    
    /usr/include/phonenumbers/phonenumber.pb.h:13:2:
         error: #error incompatible with your Protocol Buffer headers. Please update
         #error incompatible with your Protocol Buffer headers.  Please update
          ^
    
    /usr/include/phonenumbers/phonenumber.pb.h:14:2:
         error: #error your headers.
         #error your headers.
          ^
    
    /usr/include/phonenumbers/phonenumber.pb.h:22:35:
         fatal error: google/protobuf/arena.h: No such file or directory
    compilation terminated.
```

**Comment by christian-marie** [comment]
Eew, I forgot that phone numbers require protocol buffers. This isn't testing debian, is it?

I'm not sure what to do with this, is it broken for just this package, or has the distro simply broken it completely?

**Comment by christian-marie** [comment]
I mean, it looks to me that phonenumber.pb.h was generated by a later version of protocol buffers than your distro seems to ship with. Are you sure that both libphonenumber-dev and libprotobuf-dev are at the latest and consistent versions?

**Comment by jberryman** [comment]
I'm pretty sure I just did an `apt-get install libphonenumber-dev` to get this new version. I had to recover the old one from my cache. So I think you're probably right, and this is just a debian issue, though maybe it's something I did wrong. 

The package doesn't seem to be maintained very well I'm discovering, as it turns out the jessie package has been totally broken for like a year due to: https://github.com/googlei18n/libphonenumber/pull/570 https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=813630
